test/sqllogictest/transform/literal_lifting.slt

```diff
@@ -804,19 +804,18 @@ Explained Query:
         Project (#0{a}) // { arity: 1 }
           Join on=(#0{a} = #1) type=differential // { arity: 2 }
             implementation
-              %0:t_pk[#0{a}]UKiif » %1[#0]UKiif
+              %0:t_pk[#0{a}]UKiif » %1[#0]Kiif
```

Comment:
@ggevay This missing `U` may connect to some pattern detector we have, but also I'm delighted to disappoint it if so. It may also connect to having no clue about primary keys offered by the `TableFunc` logic, but similarly no real interest in enabling the optimizer around that.